examples/central-counter/frontend/bower.json

```diff
@@ -8,63 +8,70 @@
   "main": "support/index.js",
   "license": "BSD3",
   "dependencies": {
-    "purescript-pux": "^5.0.3",
-    "purescript-affjax": "^1.0.0",
-    "purescript-aff": "^1.0.0",
-    "purescript-argonaut-generic-codecs": "^3.0.0",
-    "purescript-servant-support": "^5.0.0",
-    "purescript-subscriber": "^1.0.0",
+    "purescript-pux": "^7.0.0",
+    "purescript-affjax": "^3.0.0",
+    "purescript-aff": "^2.0.0",
+    "purescript-argonaut-generic-codecs": "^4.0.0",
+    "purescript-servant-support": "^6.0.0",
+    "purescript-subscriber": "https://github.com/iilab/purescript-subscriber.git#master",
```

Comment:
That's temporary until this https://github.com/eskimor/purescript-subscriber/pull/7 is merged and published.